lib/kernel_tests.h

```diff
@@ -71,7 +71,7 @@ std::vector<volk_test_case_t> init_test_list(volk_test_params_t test_params)
     QA(VOLK_INIT_TEST(volk_16ic_x2_dot_prod_16ic, test_params))
     QA(VOLK_INIT_TEST(volk_16i_s32f_convert_32f, test_params))
     QA(VOLK_INIT_TEST(volk_16i_convert_8i, test_params))
-    QA(VOLK_INIT_TEST(volk_16i_32fc_dot_prod_32fc, test_params_inacc))
+    QA(VOLK_INIT_TEST(volk_16i_32fc_dot_prod_32fc, test_params.make_absolute(1e-1)))
```

Comment:
The `16i` input is in the range -7 .. +7 (compared to -1 .. +1 for the 32f input in the other kernel) so the tolerance needs to be larger here.

Reply:
Does it fail due to inaccuracy sometimes?